### ISSUE TITLE
[BUG] Sequoia LoginForm.qml throws an error on line 13 and prevents login

### ISSUE THREAD

**Issue by Spectranomica** [mylinuxforwork/dotfiles#398]
**Describe the bug**
Installing the new dotfiles and, more importantly, the new Sequoia theme for SDDM breaks SDDM so thoroughly that you cannot use it to login. After 5 hours of digging through logs and configs I finally found an error flag in an SDDM adjacent runtime in journalctl that shows that line 13 of the LoginForm.qml file has a syntax error preventing the login prompt itself from working. I confirmed the theme was the issue by reverting SDDM to the sugar-candy theme and everything works fine.

**To Reproduce**
1. Install the Sequoia theme
2. Reboot
3. Try to login via SDDM

**Expected behavior**
SDDM actually logs me in to my desktop environment

**Distribution (please complete the following information):**
 - Arch Linux

**Comment by mylinuxforwork** [comment]
Thanks for testing. Just installed everything today from scratch for my new video on Minimal Arch Linux without any issues. The theme worked perfectly.

But I will try again. Maybe the developer changed something.

**Comment by mylinuxforwork** [comment]
@Spectranomica Just tested it again on my laptop. No issues. All packages are updated on Arch Linux.

I have added a troubleshooting section.
https://github.com/mylinuxforwork/dotfiles/wiki/Troubleshooting

**Comment by Spectranomica** [comment]
Hmm. I have then broken the theme and that's interesting because I could not fix it by doing clean reinstalls for SDDM and ML4W but in retrospect I did not reinstall the theme. I actually did change something too. I changed line 47 of theme.conf in sequoia's install directory to "skipToLogin=true" which is exactly what the comment above it tells you to do if you want taken directly to login. Admittedly I assumed this was no different than skipping the greeting message "Welcome back!" by changing the greeting to be blank. If setting skipToLogin to true, as the comment above suggests you to do if you want to "be taken directly to the login screen" which I did want, that does still seem like an issue with the theme. It should be able to handle both cases or it should not tell you in a comment to change it into a fail state. I will do some testing to confirm this is the problem when time allows, but it is the only thing I changed anywhere in the theme's various files and subdirectories
